modules/programs/khard.nix

```
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.programs.khard;
+
+  accounts =
+    lib.filterAttrs (_: acc: acc.khard.enable) config.accounts.contact.accounts;
+
+  renderSettings = with lib.generators;
+    toINI {
+      mkKeyValue = mkKeyValueDefault rec {
+        mkValueString = v:
+          if lib.isList v then
+            lib.concatStringsSep ", " (map mkValueString v)
+          else if lib.isBool v then
+            if v then "yes" else "no"
+          else
+            v;
+      } "=";
+    };
+in {
+  meta.maintainers = [ lib.hm.maintainers.olmokramer ];
+
+  options.programs.khard = {
+    enable = lib.mkEnableOption "Khard: an address book for the Unix console";
+
+    settings = lib.mkOption {
+      type = with lib.types;
+        submodule {
+          freeformType = let
+            prim = either bool str;
+            primOrList = either prim (listOf prim);
+          in attrsOf (attrsOf primOrList);
+        };
+      default = { };
+      description = ''
+        Khard settings. See
+        <link xlink:href="https://khard.readthedocs.io/en/latest/#configuration"/>
```

```suggestion
        <https://khard.readthedocs.io/en/latest/#configuration>
```

since option descriptions now are interpreted as Markdown.